presto-kafka/src/test/java/io/prestosql/plugin/kafka/schema/confluent/KafkaWithConfluentSchemaRegistryQueryRunner.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka.schema.confluent;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.inject.Module;
+import io.airlift.log.Level;
+import io.airlift.log.Logger;
+import io.airlift.log.Logging;
+import io.prestosql.plugin.kafka.KafkaPlugin;
+import io.prestosql.plugin.kafka.KafkaQueryRunner;
+import io.prestosql.plugin.tpch.TpchPlugin;
+import io.prestosql.testing.DistributedQueryRunner;
+import io.prestosql.testing.kafka.TestingKafka;
+import io.prestosql.testing.kafka.TestingKafkaWithSchemaRegistry;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static io.airlift.testing.Closeables.closeAllSuppress;
+import static io.prestosql.plugin.kafka.KafkaPlugin.DEFAULT_EXTENSION;
+import static io.prestosql.testing.TestingSession.testSessionBuilder;
+import static java.util.Objects.requireNonNull;
+
+public class KafkaWithConfluentSchemaRegistryQueryRunner
```

Comment:
Why can't you use simply `KafkaQueryRunner`? Let's make `TestingKafkaWithSchemaRegistry ` and `TestingKafka` to share common interface (maybe interface could be called `TestingKafka` and implementation `BasicTestingKafka`, wdyt?), are there any other issues?

You can keep this class for main method and some utils if you like. BTW I like the main method here ;)